doc/users/fonts.rst

```diff
@@ -0,0 +1,79 @@
+All about Matplotlib and Fonts!
+===============================
+
+The story of fonts has been quite eventful throughout time. It involves
+contributions of tech giants such as the likes of Adobe, Apple and Microsoft.
+
+Types
+-----
+In practice, there are 3 types Matplotlib supports:
+
+.. list-table:: Type of Fonts
+   :header-rows: 1
+
+   * - Type 1 (PDF)
+     - Type 3 (PDF/PS)
+     - TrueType (PDF)
+   * - One of the oldest types, introduced by Adobe
+     - Similar to Type 1 in terms of introduction
+     - Newer than previous types, most commonly used today, introduced by Apple
+   * - They use simplified PostScript
+       (subset of full PostScript language)
+     - However, they use full PostScript language, which allows embedding
+       arbitrary code!
+       (in theory, even render fractals when rasterizing!)
+     - They include a virtual machine that can execute code!
+   * - These fonts support font hinting
+     - Do not support font hinting
+     - Hinting supported (virtual machine processes the "hints")
+   * - Expressed in pretty compact bytecode
```

Comment:
The bytecode is only in the charstrings, the container format is either ASCII-based pfa or binary pfb, both of which contain a restricted subset of PostScript, some of it encrypted, and the PostScript includes the charstrings.

Reply:
I see, I think this 'expressed in terms of' comparison can be removed?